bok_choy/page_object.py

```diff
@@ -25,7 +25,7 @@
 # Taken from https://www.owasp.org/index.php/XSS_Filter_Evasion_Cheat_Sheet#XSS_Locator.
 XSS_INJECTION = "'';!--\"<XSS>=&{()}"
 
-EXPECTED_INPUT_VALUE_FORMAT = re.compile(r'value="\'\';!--.*<xss.*{\(\)}"')
+EXPECTED_ATTRIBUTE_FORMAT = re.compile(r'\'\';!--&quot;<xss>=&amp;{\(\)}')
```

Comment:
No, because many of our common usages of the XSS injection string concatenate the string with some other part (for example, to make it unique). We would still get false positives for those. In fact, that was the reason why we had a false positive in the original 17 test failures, before this fix was made.